doc-content/shared-kie-docs/src/main/asciidoc/Workbench/General/customizing-project-preferences-proc.adoc

```diff
@@ -0,0 +1,91 @@
+[id='customizing-project-preferences-proc_{CONTEXT}']
+= Customizing project preferences
+
+A project stores assets and is part of a space. A space can hold multiple projects.
+
+For example, an organization has many departments, such as HR, Payroll, Engineering, R&D, and so on. Each department maps to a space and every department can have its own projects.
+
+You can create a new project from scratch or clone projects from an existing Git repository.
+
+.Procedure
```

Comment:
IMHO, this is a bit confusing. The above paragraph is about the possibility to create/import project and then there's "Procedure". To me, it sounds like this should be a procedure for creating/importing projects. Maybe, I would add some sentence add the end of the above paragraph like "As a user, you can customize the project settings in the business central." and then Procedure:... to show the user that the procedure is about the project settings customization

Reply:
Done. I have included a statement about customizing the projects in the introduction part.